package.json

```diff
@@ -6,13 +6,13 @@
     "type": "git",
     "url": "git://github.com/streamr-dev/streamr-client.git"
   },
-  "main": "src/index.js",
+  "main": "dist/index.js",
   "directories": {
     "test": "test"
   },
   "scripts": {
-    "build": "webpack --env dev && webpack --env prod && npm run test",
-    "dev": "webpack --progress --colors --watch --env dev",
+    "build": "NODE_ENV=production webpack",
```

Comment:
Previously, `npm build` enforced running tests. I think it's a pretty justifiable idea. For reference, many other build systems, for example `gradle` and `maven` always run tests after building too.

Reply:
Previously tests were run but not enforced (code was built no matter what the result of the tests were). It was a bug though, and not relevant now.
I personally don't think that failing tests should prevent building, but publishing. I'd rather put the running of tests into eg. pre-commit hook (now when we don't have a CI yet).
If you want to put it back to this command though, we can ofc do it before setting up a proper CI